pages/admin/index.tsx

```diff
@@ -2,13 +2,14 @@ import React from "react";
 import { NextPage } from "next";
 import ProtectedRoute from "@components/context/ProtectedRoute";
 import roleSpecificJson from "@constants/role-specific-questions.json";
-
+import ApplicationsTable from "@components/admin/ApplicationsTable";
 const memberRoles = roleSpecificJson.map(({ role }) => role);
 
 const Admin: NextPage = () => {
   return (
     <ProtectedRoute>
       <div className="container max-w-4xl px-4 mx-auto my-8">
+        <ApplicationsTable Students={[]} />
```

Comment:
Looks like the table is at the very top after Abu's changes got merged, could we move it to after the `Export CSV` button?